server/models/subscription.ts

```diff
@@ -189,14 +189,21 @@ export default (
     }
 
     // create NotificationsRead instances
-    const nReads = await Promise.all(subscribers.map(async (subscription) => {
-      // create NotificationsRead instance
-      const nRead = await models.NotificationsRead.create({
-        notification_id: notification.id,
-        subscription_id: subscription.id,
-        is_read: false
-      });
+    const nReads = await models.NotificationsRead.bulkCreate(subscribers.map((subscription) => ({
+      subscription_id: subscription.id,
+      notification_id: notification.id,
+      is_read: false
+    })), {
+      include: {
```

Comment:
After thorough testing: Subscription isn't being returned here with the newly created nRead instance. Not sure how you want to fetch (bulk) the subscriptions to append to the nReads, but the emails below are never sent bc nRead.Subscription is always undefined rn